package.js

```diff
@@ -7,17 +7,20 @@ Package.describe({
 
 Package.registerBuildPlugin({
   name: 'compileScssBatch',
-  use: ['caching-compiler@1.2.2 || 2.0.0', 'ecmascript@0.15.1'],
+  use: [
+    'caching-compiler@2.0.0-alpha300.15',
```

Comment:
I think this should still be `||`'d with 1.2 so it continues to work with Meteor 2